packages/data/abstract/polls_repository/lib/polls_repository.dart

```diff
@@ -20,42 +20,58 @@ abstract class PollsRepository {
 //--------------------------------------------------------------------------
   /// returns arithmetic mean of values of polls
```

Comment:
"arithmetic mean of values of polls" what does it mean?

Reply:
Arithmetic mean for every poll type (mood, productivity, etc) for every poll